### ISSUE TITLE
process is not defined 

### ISSUE THREAD

**Issue by psimoneau22** [felixrieseberg/React-Dropzone-Component#56]
I am trying to use this component via global script, and have added it to my html as a simple script tag

```
<html>
    <body>
       <script src="node_modules/react/dist/react.js"></script>
       <script src="node_modules/react-dom/dist/react-dom.js"></script>
       <script src="node_modules/react-dropzone-component/dist/dropzone.js"></script>
</body></html>
```

However, the dropzone component is throwing an uncaught exception under ReactPerf saying 'process is not defined':

```
measureMethods: function (object, objectName, methodNames) {
    if (process.env.NODE_ENV !== 'production') {
```

Am I not able to use this component without adding a module loader and using require/define?  The description implies that I should be able to add this using a script tag.

**Comment by felixrieseberg** [comment]
Thanks, that's good feedback. I'll take a quick look this morning, this is probably fairly easy to fix.

**Comment by felixrieseberg** [comment]
Hey, this is now fixed in 0.9.0!
